.github/workflows/test-library.yml

```
@@ -680,13 +679,16 @@ jobs:
         docker buildx inspect
         docker buildx ls
     - name: Set PROXYPY_CONTAINER_VERSION
-      run: |
-        echo "PROXYPY_CONTAINER_VERSION=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .)" > $GITHUB_ENV
+      run: >-
+        echo "PROXYPY_CONTAINER_VERSION=
+          $(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .)"
+          > $GITHUB_ENV
     - name: Build container
-      run: |
-        make container-buildx \
-          -e PROXYPY_PKG_PATH='dist/${{ needs.pre-setup.outputs.wheel-artifact-name }}' \
-          -e BUILDX_TARGET_PLATFORM='${{ matrix.targetplatform }}' \
+      run: >-
+        make container-buildx
+          -e PROXYPY_PKG_PATH=
+            'dist/${{ needs.pre-setup.outputs.wheel-artifact-name }}'
```

This concern is the same as above: you'll have the artifact name as a separate argument like `-e var= another_arg` meaning that `$var` will be set to an empty string. But if you break inside of the GHA substitution, it should be fine:
```suggestion
          -e PROXYPY_PKG_PATH='dist/${{
            needs.pre-setup.outputs.wheel-artifact-name
          }}'
```